Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.yml

```diff
@@ -164,6 +179,21 @@ script:
       name: limit
       required: false
       secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Filters only incidents in the specified status.
+      isArray: true
+      name: status
+      predefined:
```

Comment:
Since this is a list argument, pre-defined values will give the impression of a single select argument (you can try it in the cli, you will be able to choose only one option from the list). I would make it a simple text argument and supply all the options in the argument's description. @roysagi It's your call.